.travis.yml

```diff
@@ -2,28 +2,36 @@ language: android
 
 android:
   components:
+    # Update tools and then platform-tools explicitly so lint gets an updated database. Can be removed once 3.0 is out.
     - tools
     - platform-tools
-    - build-tools-25.0.1
-    - android-25
-    - extra-android-m2repository
-    - sys-img-armeabi-v7a-android-18
 
 jdk:
   - oraclejdk8
 
-before_script:
+before_install:
+  # Install SDK license so Android Gradle plugin can install deps.
+  - mkdir "$ANDROID_HOME/licenses" || true
```

Comment:
you probably know this already but I found the `-p` option pretty handy. It 1) doesn't fail when path already exists and also 2) creates intermediate directories if they don't exist.

Reply:
I knew of the intermediate directories thing but never tested the exit code. That's good to know for the future! At this point I'm copy/pasting this stuff and not even really looking at it. I'll be sure to update it on the next one and on any future copying (for which there will be many).